docs/providers/aws/guide/serverless.yml.md

```diff
@@ -25,6 +25,10 @@ service:
 
 frameworkVersion: '>=1.0.0 <2.0.0'
 
+disabledDeprecations: # Disable deprecation logs by their codes. Default is empty.
+  - code1 # Deprecation code to disable
```

Comment:
Let's use `DEP_CODE_1`  instead of `code1`, it'll be more similar to deprecation code format